controllers/operator/deployment.go

```diff
@@ -822,8 +870,31 @@ func generateManagerDeploymentObject(instance *operatorv1alpha1.Authentication,
 	return idpDeployment
 }
 
-func buildIdpVolumes(ldapCACert string, routerCertSecret string) []corev1.Volume {
-	return []corev1.Volume{
+func buildIdpVolumes(ldapCACert string, routerCertSecret string, auditSecretExists bool, required bool) []corev1.Volume {
+	auditVolume := corev1.Volume{
+		Name: IMAuditTLSVolume,
+		VolumeSource: corev1.VolumeSource{
+			Secret: &corev1.SecretVolumeSource{
+				SecretName: AuditTLSSecretName,
```

Comment:
This isn't always the Secret name anymore - the name's being defined in `platform-auth-idp` now.

Reply:
fixed